docs/guide/install-and-run.asciidoc

```diff
@@ -1,269 +1,104 @@
-environment names, sampling rates, instrumentations, metrics, and more.
-
-|===
-|*Agent configuration documentation*
-a|
-
-* {apm-go-ref-v}/configuration.html[Go Agent configuration]
-* {apm-java-ref-v}/configuration.html[Java Agent configuration]
-* {apm-dotnet-ref-v}/configuration.html[.NET Agent configuration]
-* {apm-node-ref}/configuring-the-agent.html[Node.js Agent configuration]
-* {apm-py-ref-v}/configuration.html[Python Agent configuration]
-* {apm-ruby-ref-v}/configuration.html[Ruby Agent configuration]
-* {apm-rum-ref-v}/configuration.html[RUM Agent configuration]
-|===
-
-A select number of configuration options can be changed directly in Kibana, without needing to redeploy the Agent.
-See {apm-app-ref}/agent-configuration.html[Agent configuration in Kibana] for more information.
+Select your service's language for installation instructions:
+// end::apm-agent[]
 
-[float]
-==== Configure Elastic Cloud
+--
+include::../tab-widgets/install-agents-widget.asciidoc[]
+--
 
-If you're running APM Server in Elastic cloud, you can configure your own user settings right in the Elasticsearch Service Console.
-Any changes are automatically appended to the `apm-server.yml` configuration file for your instance.
+TIP: Check the {apm-overview-ref-v}/agent-server-compatibility.html[Agent/Server compatibility matrix] to ensure you're using agents that are compatible with your version of {es}.
 
-Full details are available in the {cloud}/ec-manage-apm-settings.html[APM user settings] documentation.
 
 [float]
-==== Configure a self installation
-
-If you've installed APM Server yourself, you can edit the `apm-server.yml` configuration file to make changes.
-More information is available in {apm-server-ref-v}/configuring-howto-apm-server.html[configuring APM Server].
-
-Don't forget to also read about
-{apm-server-ref-v}/securing-apm-server.html[securing APM Server], and
-{apm-server-ref-v}/monitoring.html[monitoring APM Server].
-// end::configure-agents[]
-
-[[quick-start-overview]]
-=== Quick start development environment
+[[configure-apm]]
+=== Step 3: Advanced configuration (optional)
 
 // This tagged region is reused in the Observability docs.
-// tag::dev-environment[]
-ifeval::["{release-state}"=="unreleased"]
-
-Version {version} of APM Server has not yet been released.
-
-endif::[]
-
-ifeval::["{release-state}"!="unreleased"]
-
-If you're just looking for a quick way to try out Elastic APM, you can easily get started with Docker.
-Just follow the steps below.
-
-**Create a docker-compose.yml file**
+// tag::configure-agents[]
 
-The https://www.docker.elastic.co/[Elastic Docker registry] contains Docker images for all of the products
-in the {stack}.
-You can use Docker compose to easily get the default distributions of {es}, {kib},
-and APM Server up and running in Docker.
+Now that you're up and running with Elastic APM, you may want to adjust some configuration settings.
```

Comment:
I wonder...before I get to this part, is there some way for me to confirm that I am indeed "up and running"?

Reply:
Good point.